code/LeftAndMain.php

```diff
@@ -1660,7 +1660,11 @@ public function currentPageID()
         if (isset($this->urlParams['ID']) && is_numeric($this->urlParams['ID'])) {
             return $this->urlParams['ID'];
         }
-
+        
+        if (isset($this->getRequest()->param('ID')) && is_numeric($this->getRequest()->param('ID')) {
```

Comment:
You could remove the isset check here, because it'll return null if the key doesn't exist. There's also a syntax error in the if statement

Reply:
How about we don't do an `is_numeric` check?